test/LibraryManager.IntegrationTest/TestSolution/WebApplication1/wwwroot/js/site.js

```diff
@@ -0,0 +1 @@
+﻿// Write your Javascript code.
```

Comment:
this file and site.min.js seem wholly unnecessary.

Reply:
Have something in the wwwroot ensures the wwwroot folder will get copied to the destination folder.